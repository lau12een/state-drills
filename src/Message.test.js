import React from 'react';
import ReactDOM from 'react-dom';
import Message from './Message';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Message />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
const tree = renderer
  .create(<messages name="messages" unread={4}/>)
  .toJSON();
expect(tree).toMatchSnapshot();  
});