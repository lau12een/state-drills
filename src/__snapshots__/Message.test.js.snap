// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<messages
  name="messages"
  unread={4}
/>
`;
